### PR TITLE
added mapping =  argument before aes() per issue #473

### PR DESCRIPTION
**Pull request by mkardish**
In response to issue #473 I have implemented changes in the code to all instances of aes() in the rmd files of the ggplot2 lesson. https://github.com/swcarpentry/r-novice-gapminder/issues/473

### CODE REVIEW

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 1]
```diff
@@ -188,8 +188,8 @@ lines.
 > > happened?
 > >
 > > ```{r ch3-sol}
-> > ggplot(data = gapminder, aes(x=year, y=lifeExp, by=country)) +
-> >  geom_point() + geom_line(aes(color=continent))
+> > ggplot(data = gapminder, mapping = aes(x=year, y=lifeExp, by=country)) +
+> >  geom_point() + geom_line(mappin = aes(color=continent))
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd]: Replace `mappin` with `mapping`

**Comment by jcoliver** [comment]
Thanks, @mkardish ! If you could update one line (192) with `mappin` -> `mapping` in the pull request, we can merge this.

**Comment by mkardish** [comment]
@jcoliver updated! Sorry about that!

**Comment by jcoliver** [comment]
Thanks again, @mkardish

**Pull request merged**